help-beta/src/components/NavigationSteps.astro

```diff
@@ -320,8 +320,9 @@ const getSettingsHTML = (
     
     const relativeLink = `<a href="${setting_link}">${setting_name}</a>`;
 
-    // The "Bots" label appears in both Personal and Organization settings
-    // in the user interface so we need special text for this setting.
+    // The "Bots" and the "Users" label appears in both Personal and
```

Comment:
There isn't a "Users" tab in the personal settings overlay. For that case, the users tab of the organization settings overlay has different panels (active, deactivated, invitations), so we use the alternative instructions for that case too.

Reply:
[Fixed](https://github.com/zulip/zulip/compare/19bf2816bd1d1eec15778b18049fb6a0c7a7fa43..e6ecc7572a36cab3a0e3d8194c9345c0bc6999ba)